pandas/tseries/tools.py

```diff
@@ -267,7 +267,7 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     If a date that does not meet timestamp limitations, passing errors='coerce'
     will force to NaT. Furthermore this will force non-dates to NaT as well.
 
-    >>> pd.to_datetime('13000101', format='%Y%m%d')
```

Comment:
it's nice to add an example of default behavior, and describe what the timestamp limitations is,